Sources/swift-package-collection/Entrypoint.swift

```diff
@@ -13,4 +13,10 @@
 import Commands
 import PackageCollectionsTool
 
-SwiftPackageCollectionsTool.main()
+@main
```

Comment:
This will require `-parse-as-library` added to `CMakeLists.txt` in this directory, and the name of the file to be updated there as well, otherwise bootstrapping build will fail.

Reply:
Copy-paste here should be fine, thanks!